subsys/debug/coredump/Kconfig

```diff
@@ -28,6 +28,13 @@ config DEBUG_COREDUMP_BACKEND_FLASH_PARTITION
 	  Core dump is saved to a flash partition with DTS alias
 	  "coredump-partition".
 
+config DEBUG_COREDUMP_MW_BACKEND_INTEL_ADSP
+	bool "Use memory window 2 slot [1] space for coredump"
```

Comment:
Maybe `Use memory window for coredump on Intel Audio DSP` as this is specific to Intel Audio DSPs.
IMHO, there is no need to specify which memory window and which slot to use. This is implementation details that does not need to encoded in kconfig summary and help texts. Also, which window and slot to use may change in the future.

Reply:
Done